scripts/eurostat/birth_death_migration/Readme.md

```diff
@@ -0,0 +1,55 @@
+# Importing Eurostat Demographic balance and crude rates into Data Commons
+
+Author: jinpz
+
+## Table of Contents
+
+1. [About the Dataset](#about-the-dataset)
+1. [Import Procedure](#Import-Procedure)
```

Comment:
This link is broken, maybe bc of caps?

Reply:
just tested it on my fork, it seems to be okay. For consistency, I replaced the caps. Where would it break?